AzureCore/Source/AzureKeys.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  AzureKeys.swift
+//  AzureCore
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import Foundation
+
+public class AzureKeys: Codable {
+    enum CodingKeys: String, CodingKey {
+        case accountName = "AzureAccountName"
+        case masterKey = "AzureAccountMasterKey"
+    }
+
+    public let accountName: String
+    public let masterKey: String
+
+    public var hasValidAccountName: Bool {
+        return !accountName.trimmingCharacters(in: .whitespaces).isEmpty
+    }
+
+    public var hasValidMasterKey: Bool {
```

Comment:
We should be checking for default values like [here](https://github.com/colbylwilliams/CustomVision/blob/master/CustomVision/CustomVision/CustomVisionKeys.swift)

Reply:
Also, let's add a Azure.plist file with the default values.  Then we can add a custom `AzureTests.plist` file, use the `configure` override that takes the plist name, and ignore the `AzureTests.plist`